java/src/jmri/web/servlet/about/AboutServlet.java

```diff
@@ -38,23 +38,23 @@ protected void processRequest(HttpServletRequest request, HttpServletResponse re
                 connList += comma + Bundle.getMessage(request.getLocale(), "ConnectionSucceeded", conn.getConnectionName(), conn.name(), conn.getInfo());
                 comma = ", ";
             }
-        }     
+        }
 
         //print the html, using the replacement values listed to fill in the calculated stuff
         response.setHeader("Connection", "Keep-Alive"); // NOI18N
         response.setContentType(UTF8_TEXT_HTML);
         response.getWriter().print(String.format(request.getLocale(),
                 FileUtil.readURL(FileUtil.findURL(Bundle.getMessage(request.getLocale(), "About.html"))),
-                Bundle.getMessage(request.getLocale(), "AboutTitle"),               //page title is parm 1
-                ServletUtil.getInstance().getNavBar(request.getLocale(), "/about"), //navbar is parm 2
-                ServletUtil.getInstance().getRailroadName(false),                   //railroad name is parm 3
-                ServletUtil.getInstance().getFooter(request.getLocale(), "/about"), //footer is parm 4
-                connList,                                            //system connection list is parm 5
-                Application.getApplicationName() + " " + jmri.Version.name(),       //JMRI version is parm 6                                         //JMRI version is parm 6
-                jmri.Version.getCopyright(),                                        //Copyright is parm 7
-                System.getProperty("java.version", "<unknown>"),                    //Java version is parm 8
```

Comment:
I think the "auto-remove redundant white space" approach has reduced the readability of this section. The aligned comments (older version) communicate better. Not sure what tool you're using to do this, but can it be made a bit smarter?